kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -70,6 +70,52 @@ def kprove_args(self) -> ArgumentParser:
         )
         return args
 
+    @cached_property
+    def kprove_legacy_args(self) -> ArgumentParser:
+        args = ArgumentParser(add_help=False)
+        args.add_argument(
+            '--bug-report',
+            default=False,
+            action='store_true',
+            help='Generate a haskell-backend bug report for the execution.',
+        )
+        args.add_argument(
+            '--debugger',
+            dest='debugger',
+            default=False,
+            action='store_true',
+            help='Launch proof in an interactive debugger.',
+        )
```

Comment:
Did you havea  chance to check if `--debugger` is working as expected? Main concern is that launching the `kore-repl` from Python may behave weird in terms of input/output?

Reply:
Good catch.

Actually, `--debugger` doesn't work at all currently because it can't be used with `--haskell-backend-command`, so I'll have to revert runtimeverification/pyk/pull/451. 

After fixing this issue though (runtimeverification/pyk/pull/455), `--debugger` seems to work fine and there's no problem with the I/O.